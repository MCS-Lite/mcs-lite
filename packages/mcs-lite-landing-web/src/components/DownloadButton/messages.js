import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'DownloadButton.downloadFor': {
    id: 'DownloadButton.downloadFor',
    defaultMessage: 'Download for',
  },
  'DownloadButton.or': {
    id: 'DownloadButton.or',
    defaultMessage: 'or',
  },
  'DownloadButton.others': {
    id: 'DownloadButton.others',
    defaultMessage: 'Download for other platforms',
  },
});

export default messages;
