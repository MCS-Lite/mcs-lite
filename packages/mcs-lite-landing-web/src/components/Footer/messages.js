import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'Footer.desc': {
    id: 'Footer.desc',
    defaultMessage: 'MediaTek Inc. All Rights Reserved.',
  },
});

export default messages;
