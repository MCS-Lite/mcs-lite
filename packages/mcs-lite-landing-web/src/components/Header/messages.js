import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'Header.gotoMCS': {
    id: 'Header.gotoMCS',
    defaultMessage: 'Go to MCS',
  },
  'Header.resource': {
    id: 'Header.resource',
    defaultMessage: 'Resources',
  },
  'Header.contact': {
    id: 'Header.contact',
    defaultMessage: 'Contact us',
  },
});

export default messages;
