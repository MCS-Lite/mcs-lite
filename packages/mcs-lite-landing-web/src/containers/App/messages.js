import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'App.welcome': {
    id: 'App.welcome',
    defaultMessage: 'Welcome',
  },
});

export default messages;
