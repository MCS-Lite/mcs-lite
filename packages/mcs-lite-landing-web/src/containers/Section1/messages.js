import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'Section1.title': {
    id: 'Section1.title',
    defaultMessage: 'Build your IoT platform with ease and flexibility',
  },
  'Section1.desc': {
    id: 'Section1.desc',
    defaultMessage:
      'MCS Lite, inspired from features of MCS online with easy-to-use user interface, is designed for users who desire dedicate IoT platform for their applications. Binaries are available in Windows, Linux and Mac OS for instant launch.',
  },
});

export default messages;
