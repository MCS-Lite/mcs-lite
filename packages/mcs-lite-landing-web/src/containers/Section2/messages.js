import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'Section2.title': {
    id: 'Section2.title',
    defaultMessage: 'Launch MCS Lite in seconds',
  },
  'Section2.desc': {
    id: 'Section2.desc',
    defaultMessage:
      'MCS Lite binary can be launched on your computer in one-click, you can also deploy on-premise or in the cloud with provided source code. All the options are here for your IoT applications during prototype, development and production stages.',
  },
});

export default messages;
