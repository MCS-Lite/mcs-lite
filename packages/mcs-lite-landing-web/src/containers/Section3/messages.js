import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'Section3.title': {
    id: 'Section3.title',
    defaultMessage: 'Explore built-in features',
  },
  'Section3.desc': {
    id: 'Section3.desc',
    defaultMessage:
      'Start to collect data from your IoT devices and control them anytime, anywhere.',
  },

  'Section3.data.title': {
    id: 'Section3.data.title',
    defaultMessage: 'Data collection',
  },
  'Section3.data.desc': {
    id: 'Section3.data.desc',
    defaultMessage: 'Collect and visualize data from your devices',
  },
  'Section3.remote.title': {
    id: 'Section3.remote.title',
    defaultMessage: 'Remote control',
  },
  'Section3.remote.desc': {
    id: 'Section3.remote.desc',
    defaultMessage: 'Control your devices from MCS Lite console',
  },
  'Section3.management.title': {
    id: 'Section3.management.title',
    defaultMessage: 'Management',
  },
  'Section3.management.desc': {
    id: 'Section3.management.desc',
    defaultMessage: 'Manage all your connected devices',
  },
  'Section3.mobile.title': {
    id: 'Section3.mobile.title',
    defaultMessage: 'Mobile friendly',
  },
  'Section3.mobile.desc': {
    id: 'Section3.mobile.desc',
    defaultMessage:
      'Responsive Web Design for easy access on your mobile and tablet',
  },
});

export default messages;
