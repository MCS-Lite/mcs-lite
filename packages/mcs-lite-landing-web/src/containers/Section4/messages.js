import { defineMessages } from 'react-intl';

const messages = defineMessages({
  'Section4.title': {
    id: 'Section4.title',
    defaultMessage: 'Extend and get creative with MCS Lite',
  },
  'Section4.github.title': {
    id: 'Section4.github.title',
    defaultMessage: 'Open Source',
  },
  'Section4.github.desc': {
    id: 'Section4.github.desc',
    defaultMessage:
      'MCSLite is distributed as an open-source software under MIT license. You can use, copy, modify, merge, publish, distribute, sublicense our projects.',
  },
  'Section4.github.button': {
    id: 'Section4.github.button',
    defaultMessage: 'Follow us on GitHub',
  },
  'Section4.custom.title': {
    id: 'Section4.custom.title',
    defaultMessage: 'Customization',
  },
  'Section4.custom.desc': {
    id: 'Section4.custom.desc',
    defaultMessage:
      'Whether you want to customize the user interface, support other databases or add additional data analysis and encryption mechanism, you can directly modify the source code or use the API provided by MCS Lite to expand the functions.',
  },
  'Section4.custom.button': {
    id: 'Section4.custom.button',
    defaultMessage: 'Customize your UI',
  },
});

export default messages;
