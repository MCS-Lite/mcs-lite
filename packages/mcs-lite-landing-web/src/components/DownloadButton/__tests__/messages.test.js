import messages from '../messages';

it('should return messages', () => {
  expect(messages).toMatchSnapshot();
});
