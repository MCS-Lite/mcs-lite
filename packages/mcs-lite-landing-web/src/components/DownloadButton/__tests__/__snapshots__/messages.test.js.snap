// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return messages 1`] = `
Object {
  "DownloadButton.downloadFor": Object {
    "defaultMessage": "Download for",
    "id": "DownloadButton.downloadFor",
  },
  "DownloadButton.or": Object {
    "defaultMessage": "or",
    "id": "DownloadButton.or",
  },
  "DownloadButton.others": Object {
    "defaultMessage": "Download for other platforms",
    "id": "DownloadButton.others",
  },
}
`;
